src/app/components/JazzIcon/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { memo, useEffect, useRef } from 'react'
+import jazzicon from '@metamask/jazzicon'
+import { Box } from 'grommet'
+
+interface JazzIconProps {
+  diameter: number
+  seed: number
+}
+
+export const JazzIcon = memo(({ diameter, seed }: JazzIconProps) => {
+  const ref = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    if (ref?.current) {
+      const icon = jazzicon(diameter, seed)
+
+      ref.current.replaceChildren()
+      ref.current.appendChild(icon)
```

Comment:
Is this preferred over `ref.current.replaceChildren(icon)`?

Reply:
nah that's my oversight